src/processing/former/former.md

```diff
@@ -0,0 +1,604 @@
+---
+title: Former
+level: 1
+logo: ../../assets/img/ccuk_logo.png
+author: Sigmund Hansen
+---
+
+# Introduksjon: {.intro}
+
+I denne modulen skal vi lære et programmeringsspråk og verktøy som
+heter Processing. Det ble laget for å gjøre programmering lett for
+designere og andre som ikke har programmert før, sånn at de kan lage
+multimedieprogrammer, spill og liknende. Mange liker Processing fordi
+det går fort å lage programmer som vanligvis krever mye arbeid.
+
+# Steg 1: Vindu {.activity}
+
+Nå skal vi begynne helt enkelt med å lage et vindu og fylle det med en
+bakgrunnsfarge. Dette vil bli grunnlaget for nesten alle programmer
+som du lager med Processing, så det er et fint sted å starte.
+
+## Sjekkliste {.check}
+
++ Start Processing og skriv dette:
+
+    ```processing
+    void setup() {
+      size(640, 480);
+    }
+    
+    void draw() {
+      background(0);
+    }
+    ```
++ Kjør programmet ved å trykke på **Ctrl+R**, knappen med en pil
+  eller **Sketch --> Run** i menyen.
++ Lagre programmet som Former ved å trykke på **Ctrl+S** eller
+  velg **File --> Save** i menyen.
+
+### Hvordan skrive krøllparenteser og liknende {.protip}
+
+TODO: Fikse bredden på celler i tabellen (det trengs marger for å få
+litt horisontal luft). Sjekk at pipe skrives med Shift + 7 på Mac.

```

Comment:
Det er nok Option + 7. Cmd brukes vel ikke for å skrive tegn, men jeg må låne Macen til kjæresten min og teste det. Det er i hvert fall ikke Shift + 7 som det står nå.